tests/support/pkg.py

```diff
@@ -295,6 +295,10 @@ def __attrs_post_init__(self):
                         self.run_root = self.bin_dir / "run"
                     else:
                         log.error("Unexpected file extension: %s", self.file_ext)
+                log.debug("root: %s", self.root)
+                log.debug("bin_dir: %s", self.bin_dir)
+                log.debug("ssm_bin: %s", self.ssm_bin)
+                log.debug("run_root: %s", self.run_root)
```

Comment:
```suggestion
                log.debug("root: %s", self.root)
                log.debug("bin_dir: %s", self.bin_dir)
                log.debug("ssm_bin: %s", self.ssm_bin)
                log.debug("run_root: %s", self.run_root)
```
Could be done in a single log call, not actually asking to change since this in on tests and not salt.